pkgs/development/r-modules/default.nix

```
@@ -1402,6 +1402,10 @@ let
       ];
     });
 
+    covidsymptom = old.covidsymptom.overrideAttrs (attrs: {
+      patches = [ ./patches/covidsymptom.patch ];
```

```suggestion
      preConfigure = "rm R/covidsymptomdata.R";
```
Would this work? It's smaller than bundling a patch.

---

:facepalm: